modules/api/src/main/scala/vinyldns/api/domain/ReverseZoneHelpers.scala

```diff
@@ -62,7 +63,7 @@ object ReverseZoneHelpers {
     }
 
   def convertPTRtoIPv4(zone: Zone, recordName: String): String = {
-    val zoneName = zone.name.split("in-addr.arpa.")(0)
+    val zoneName = Pattern.compile("in-addr.arpa.", Pattern.CASE_INSENSITIVE).split(zone.name)(0)
```

Comment:
This is not idiomatic Scala. We want to try to avoid pulling in Java libraries. To do this in Scala, you would use the `.r` function:

```scala
    val zoneName = "(?i)in-addr.arpa.".r.split(zone.name)(0)
```

However, this seems like overkill when we're just trying to remove the suffix. Instead, we can do this:
```suggestion
    val zoneName = zone.name.dropRight("in-addr.arpa.".length)
```

Reply:
Updated the code as suggested.